### ISSUE TITLE
Sort by date created or date modified

### ISSUE THREAD

**Issue by graydon** [graydon/obsidian-dangling-links#8]
Currently the sort order for dangling items is only alpha ascending or descending. As requested in #1 it might be nice to handle date sorts as well.

**Comment by flip-md** [comment]
I would take this one in the next days unless you want to.

**Comment by graydon** [comment]
Nah, go for it -- I'm totally swamped with other stuff right now.

**Comment by claremacrae** [comment]
> I would take this one in the next days unless you want to.

Hi @flip-md, I know this was a long time ago, but just to say that if you were still interested in doing this, I would be happy to review and merge a PR.

**Comment by flip-md** [comment]
> > I would take this one in the next days unless you want to.
> 
> Hi @flip-md, I know this was a long time ago, but just to say that if you were still interested in doing this, I would be happy to review and merge a PR.

Hey @claremacrae I do not use Obsidian anymore and am quite swamped with other work so far, so if anyone else is interested in taking it, feel free to!

**Comment by claremacrae** [comment]
Hi @flip-md - understood - many thanks for the swift reply.
